.circleci/config.yml

```diff
@@ -28,6 +28,10 @@ workflows:
           requires:
             - godeps
             - jsdeps
+      - cross_build:
+          requires:
+            - godeps
+            - jsdeps
```

Comment:
I am torn between running this on every PR (catches problems early, but slow) and only running it on `master`. Thoughts?

Reply:
Better I think to weed out problems early.